library/core/resource_loader/src/main/java/org/quiltmc/qsl/resource/loader/api/ResourceLoader.java

```diff
@@ -67,19 +69,50 @@ static ResourceLoader get(ResourceType type) {
 	 * @param activationType the activation type of the resource pack
 	 * @return {@code true} if successfully registered the resource pack, otherwise {@code false}
 	 * @throws IllegalArgumentException if a mod with the corresponding namespace given in id cannot be found
+	 * @see #registerBuiltinResourcePack(Identifier, ResourcePackActivationType, Text)
 	 * @see #registerBuiltinResourcePack(Identifier, ModContainer, ResourcePackActivationType)
+	 * @see #registerBuiltinResourcePack(Identifier, ModContainer, ResourcePackActivationType, Text)
 	 */
 	static boolean registerBuiltinResourcePack(Identifier id, ResourcePackActivationType activationType) {
+		return registerBuiltinResourcePack(id, activationType, ResourceLoaderImpl.getBuiltinPackDisplayNameFromId(id));
+	}
+
+	/**
+	 * Registers a built-in resource pack.
+	 * <p>
+	 * A built-in resource pack is an extra resource pack provided by your mod which is not always active,
+	 * it's similar to the "Programmer Art" resource pack.
+	 * <p>
+	 * Why and when to use it?
```

Comment:
This could be formatted as a header instead, but I think it makes more sense to just remove it.
```suggestion
```